core/consensus/authority/authority_manager.hpp

```diff
@@ -19,7 +19,7 @@ namespace kagome::authority {
      * @return outcome authority set
      */
     virtual outcome::result<std::shared_ptr<const primitives::AuthorityList>>
-    authorities(const primitives::BlockInfo &block) = 0;
+    authorities(const primitives::BlockInfo &block, bool finalized) = 0;
```

Comment:
Not quite clear what finalized means here.

Reply:
Comment will be appended